core/errors.go

```diff
@@ -29,6 +30,8 @@ func isTemporary(info httperror.Info, err error) bool {
 		return true
 	case "CH001": // request timed out
 		return true
+	case "CH012": // conflict
+		return true
```

Comment:
Yeah, the idea was latter. We could try to handle it in the server, but to-date we've generally pushed that retry logic into the clients to avoid multiplying our retries by retrying at different points in the request cycle. This error seems akin to a Postgres serialization conflict error. Does lib/pq automatically retry that for us?